### PR TITLE
[KOGITO-2535] Fix ImageStream name when custom image is given

### PR DESCRIPTION
**Pull request by ricardozanini**
See: https://issues.redhat.com/browse/KOGITO-2535

Small fix to a bug introduced by the `KogitoBuild` feature.

Signed-off-by: Ricardo Zanini <zanini@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml [hunk 1]
```diff
@@ -391,6 +391,11 @@ spec:
                 - name
                 type: object
               type: array
+            httpPort:
+              description: HttpPort will set the environment env HTTP_PORT to define
```
Comment (ricardozanini) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml:395]: [People, for god's sake, please run `make test` before sending a PR!](https://github.com/kiegroup/kogito-cloud-operator#kogito-operator-collaboration-and-pull-requests)

Comment (ricardozanini) [reply deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml:395]: @Kaitou786 do you believe you can do the same thing in the CI check we did for `fmt`, but now on `operator-sdk generate crds|k8s` and `openapi gen`? This way we can warn contributors when subimmiting a PR.

Comment (Kaitou786) [reply deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml:395]: @ricardozanini definitely I'll add these checks  :-)

**Comment by sutaakar** [comment]
All the tests passed.

**Pull request merged**